search/tests/stac/catalog.spec.js

```diff
@@ -50,4 +50,11 @@ describe('createRootCatalog', () => {
     expect(childLink).toBeDefined();
     expect(childLink.href).toBe('/cmr-stac/stac/default');
   });
+
+  it('should be able to add a next rel to pagination links', () => {
+    rootCatalog.addChild('Pagination Link', '/provider');
+    const paginationLink = rootCatalog.links.find((link) => link.rel === 'child');
+    expect(paginationLink).toBeDefined();
+    expect(paginationLink.href).toBe('/cmr-stac/stac/provider');
```

Comment:
We might want to add a test in here to make sure that the relation is 'next'. It could be as simple as 
`expect(paginationLink.rel).toEqual('next');`

Reply:
Done.